includes/managers/widgets.php

```diff
@@ -106,11 +105,13 @@ private function init_widgets() {
 		$this->register_promoted_widgets();
 
 		foreach ( $build_widgets_filename as $widget_filename ) {
-			$this->register_widget_by_filename( $widget_filename );
-		}
+			include ELEMENTOR_PATH . 'includes/widgets/' . $widget_filename . '.php';
```

Comment:
I think we should put the atomic-widgets in a dedicated folder.

Reply:
They are, I rolled back this file to its original state.